dashboard/db/migrate/20190711180756_add_seen_to_teacher_feedbacks.rb

```diff
@@ -0,0 +1,6 @@
+class AddSeenToTeacherFeedbacks < ActiveRecord::Migration[5.0]
+  def change
+    add_column :teacher_feedbacks, :seen, :boolean, default: false
```

Comment:
is there a specific reason we need this column? i think we could achieve the same behavior with just the `seen_at` column. if the column is null, `seen` will be false, otherwise it will be true

Reply:
Good point. I'm going to think on this in conjunction with the info we need about whether it was seen on the feedback overview page or on the level and revise.

Reply:
Ok I finally wrapped my brain around this. Does this change and the updated description look right? Thanks for the feedback!

Reply:
yes! this looks great. thank you for updating!